packages/api/src/get-test.js

```diff
@@ -0,0 +1,66 @@
+
+// Copyright 2020 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+
+// class to receive POSTS with build information
+
+const firebaseEncode = require('../lib/firebase-encode');
+
+class GetTestHandler {
+  constructor (app, client) {
+    this.app = app;
+    this.client = client;
+  }
+
+  listen () {
+    // must use query parameter for test name since irregular names
+    //      required query param: name
+    //      optional query params: limit (# of builds), tag, matrix, os, ref
+    this.app.get('/api/repo/:orgname/:reponame/test', async (req, res, next) => {
+      try {
+        const repoid = req.params.orgname + '/' + req.params.reponame;
+        if (!req.query.name) {
+          throw new Error('Route requires query parameter of name');
+        }
+        const name = req.query.name;
+        let limit = 30;
```

Comment:
```suggestion
        const limit = 30;
```

Reply:
This variable is conditionally changed if limit is passed in as a parameter